pkg/client/cmd/app.go

```diff
@@ -423,6 +425,90 @@ WARNING:
 	Run: appLogs,
 }
 
+var appAutoScaleSetCmd = &cobra.Command{
+	Use:   "autoscale <name> [flags]",
+	Short: "Set autoscale parameters for the app",
+	Long: `Set auto scaling parameters for the application.
+
+You can configure the autoscale parameters for the application. (i.e. the minimum and maximum number of replicas)
+
+	Example:   To set the number minimum of replicas to 2:
+
+  $ teresa app autoscale myapp --min 2`,
+	Run: appAutoScaleSet,
+}
+
+func appAutoScaleSet(cmd *cobra.Command, args []string) {
+	if len(args) != 1 {
+		cmd.Usage()
+		return
+	}
+
+	name := args[0]
+
+	min, err := cmd.Flags().GetInt32("min")
+	if err != nil {
+		client.PrintErrorAndExit("invalid min parameter")
+	}
+
+	max, err := cmd.Flags().GetInt32("max")
+	if err != nil {
+		client.PrintErrorAndExit("invalid max parameter")
+	}
+
+	cpu, err := cmd.Flags().GetInt32("cpu-percent")
+	if err != nil {
+		client.PrintErrorAndExit("invalid cpu-percent parameter")
+	}
+
+	if msg, isValid := validateFlags(min, max); isValid != true {
+		client.PrintErrorAndExit(msg)
+	}
+
+	conn, err := connection.New(cfgFile)
+	if err != nil {
+		client.PrintErrorAndExit("Error connecting to server: %s", err)
+	}
+	defer conn.Close()
+
+	as := &appb.SetAutoScaleRequest_AutoScale{
+		Min:                  min,
+		Max:                  max,
+		CpuTargetUtilization: cpu,
+	}
+	req := &appb.SetAutoScaleRequest{
+		Name:      name,
+		AutoScale: as,
+	}
+	cli := appb.NewAppClient(conn)
+	if _, err := cli.SetAutoScale(context.Background(), req); err != nil {
+		client.PrintErrorAndExit(client.GetErrorMsg(err))
+	}
+	fmt.Println("Autoscale updated with success")
+}
+
+func validateFlags(min, max int32) (msg string, isValid bool) {
```

Comment:
What do you think about don't use named return values?
I'm thinking about this func and sounds like unnecessary.
It's simple to write something like this:
```go
if max == flagNotDefined || flagNotDefined == {
    return "--min and --max are required`, false
}
```
With this change you don't need to use `else` f.ex/